services/bundle_analysis/new_notify/contexts/comment.py

```diff
@@ -0,0 +1,156 @@
+import logging
+from typing import Literal
+
+from asgiref.sync import async_to_sync
+from shared.bundle_analysis import (
+    BundleAnalysisComparison,
+)
+from shared.yaml import UserYaml
+
+from database.models.core import Commit
+from services.bundle_analysis.comparison import ComparisonLoader
+from services.bundle_analysis.exceptions import (
+    MissingBaseCommit,
+    MissingBaseReport,
+    MissingHeadCommit,
+    MissingHeadReport,
+)
+from services.bundle_analysis.new_notify.contexts import (
+    BaseBundleAnalysisNotificationContext,
+    NotificationContextBuilder,
+    NotificationContextBuildError,
+)
+from services.bundle_analysis.new_notify.types import NotificationType
+from services.repository import (
+    EnrichedPull,
+    fetch_and_update_pull_request_information_from_commit,
+)
+
+log = logging.getLogger(__name__)
+
+
+class BundleAnalysisCommentNotificationContext(BaseBundleAnalysisNotificationContext):
+    """Context for the Bundle Analysis PR Comment. Extends BaseBundleAnalysisNotificationContext."""
+
+    notification_type = NotificationType.PR_COMMENT
+
+    @property
+    def pull(self) -> EnrichedPull:
```

Comment:
Same comment about empty property handling if applicable. I'm not sure if the builder pattern "handles' this question by virtue handling it in the builder class itself